### PR TITLE
install pip package with `--user` option

### PR DESCRIPTION
**Pull request by zoofyiscool**

### CODE REVIEW

File: install-linux.sh [hunk 1]
```diff
@@ -43,7 +43,7 @@ if [[ $(pip3 list | grep pypresence) ]]; then
 echo "[${NAME}] │   │   └── 'pypresence' has been installed"
 else
 echo "[${NAME}] │   │   └── installing 'pypresence'"
-    pip3 install pypresence
+    pip3 install pypresence --user
```
Comment (cniw) [inline install-linux.sh:46]: please move the --user between install and pypresence

**Review by cniw — CHANGES_REQUESTED** [review]
and do change on install-osx.sh file as well

**Comment by zoofyiscool** [comment]
added in new commit to the same PR

**Review by cniw — APPROVED** [review]
looks good, thanks

**Pull request merged**